packages/forklift-console-plugin/src/modules/Providers/views/details/components/DetailsSection/OVADetailsSection.tsx

```diff
@@ -42,7 +42,7 @@ export const OVADetailsSection: React.FC<DetailsSectionProps> = ({ data }) => {
         helpContent={
           <Text>
             {t(
-              'Specify the type of source provider. Allowed values are ova, ovirt, vsphere, and openstack. This label is needed to verify the credentials are correct when the remote system is accessible and, for RHV, to retrieve the Manager CA certificate when a third-party certificate is specified.',
+              'Specify the type of source provider. Allowed values are ova, ovirt, vsphere, OpenShift, and openstack. This label is needed to verify the credentials are correct when the remote system is accessible and, for RHV, to retrieve the Manager CA certificate when a third-party certificate is specified.',
```

Comment:
```suggestion
              'Specify the type of source provider. Allowed values are ova, ovirt, vsphere, openshift, and openstack. This label is needed to verify the credentials are correct when the remote system is accessible and, for RHV, to retrieve the Manager CA certificate when a third-party certificate is specified.',
```

the allowed values are lowercase
https://github.com/kubev2v/forklift-console-plugin/blob/e77a33b657037ee0fc1463035e8a28cb25752606/packages/types/src/types/ProviderSecret.ts#L11

original text:
https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.5/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#CO2-2